pkg/cluster/ports/ports.go

```diff
@@ -25,14 +25,9 @@ const (
 	// KubeletPort is the default port for the kubelet server on each host machine.
 	// May be overridden by a flag at startup.
 	KubeletPort = 10250
-	// KubeletReadOnlyPort exposes basic read-only services from the kubelet.
-	// May be overridden by a flag at startup.
-	// This is necessary for heapster to collect monitoring stats from the kubelet
-	// until heapster can transition to using the SSL endpoint.
-	// TODO(roberthbailey): Remove this once we have a better solution for heapster.
-	KubeletReadOnlyPort = 10255
-	// KubeletHealthzPort exposes a healthz endpoint from the kubelet.
-	// May be overridden by a flag at startup.
+	// LegacyKubeletReadOnlyPort is deprecated, and was exposed basic read-only services from the kubelet.
+	LegacyKubeletReadOnlyPort = 10255
```

Comment:
let's move to test where it is used.

Reply:
Move to `test/e2e/network/firewall.go` where it is used.